### ISSUE TITLE
Blindscan Plugin - Needs Additional Supported Chips by blindscan-s2 Added

### ISSUE THREAD

**Issue by ilikenwf** [OpenPLi/enigma2-plugins#997]
In my case, adding Si2169D to the list for my Zgemma...

```
_blindscans2Nims = ('TBS-5925', 'DVBS2BOX', 'M88DS3103', 'Si2169D', 'Si2169')
```

But the project should already have a good idea in a makefile somewhere that dictates which tuners support blindscan-s2 since it gets built for them, so why not add them to the above?

**Comment by WanWizard** [comment]
@Dima73 ?

**Comment by ilikenwf** [comment]
Though to be fair no matter what settings I use, I usually end up finding nothing with this method which is odd...but that is less relevant to the issue of any tuner supporting the use of blindscan-s2 should be added to that list.

**Comment by Dima73** [comment]
if n.canBeCompatible("DVB-S") and n.description.startswith("Si216"):
				have_Support_Blindscan = True

**Comment by ilikenwf** [comment]
Yes but it doesn't actually run without being added to the above "supported" list. It worked on the older openatv build the device came with, though, for what it's worth.

It is an S2/S2X receiver, in case the reporting is wrong or the kernel driver has changed how it reports the name. I can check on that if you can tell me where the device name is listed, somewhere in /dev I assume?

**Comment by Dima73** [comment]
` cat /proc/bus/nim_sockets`

**Comment by ilikenwf** [comment]
```
NIM Socket 0:
	Type: DVB-S2X
	Name: Si2169D
	Has_Outputs: no
	Frontend_Device: 0
	I2C_Device: 4
	Supports_Blind_Scan: yes
NIM Socket 1:
	Type: DVB-S2X
	Name: Si2169D
	Has_Outputs: no
	Frontend_Device: 1
	I2C_Device: 4
	Supports_Blind_Scan: yes
NIM Socket 2:
	Type: DVB-T2
	Name: Si2168
	Has_Outputs: yes
	Frontend_Device: 2
	I2C_Device: 3
	Supports_Blind_Scan: yes
```

So it does support it...but yeah without the mods I got the not supported message.

**Comment by ilikenwf** [comment]
@Dima73  https://github.com/oe-alliance/oe-alliance-plugins/commit/c5b785eb1ea102b95f6c1513821232706ce2bcbf

oe-alliance already did this at some point themselves.

**Comment by ilikenwf** [comment]
The only issue is that upon running it it looks for really off the wall frequencies like 4260V which isn't valid.

**Comment by Dima73** [comment]
> @Dima73 [oe-alliance/oe-alliance-plugins@c5b785e](https://github.com/oe-alliance/oe-alliance-plugins/commit/c5b785eb1ea102b95f6c1513821232706ce2bcbf)
> 
> oe-alliance already did this at some point themselves.

self.auto_scan = nimmanager.nim_slots[self.feid].supportsBlindScan() or tunername.startswith('Si216')
https://github.com/OpenPLi/enigma2-plugins/blob/python3/blindscan/src/dmmBlindScan.py#L473
Isn't it the same?

**Comment by Dima73** [comment]
Open plugin 
menu "Utility from the manufacturer"/"Hardware type"
use "Hardware type"

**Comment by ilikenwf** [comment]
I have used both, neither work on any given satellite. I'm currently parked and locked on 99.2W both with C and Ku band and neither tuner (instead of diseqc I have each LNB on a separate tuner for now), nothing is found, and the frequencies that are investigated seem outlandish.

**Comment by ilikenwf** [comment]
![image](https://github.com/OpenPLi/enigma2-plugins/assets/414778/ce1d3b9e-4596-4f50-ba69-c1019541c437)

**Comment by ilikenwf** [comment]
What the heck is 4260V? It shouldn't be possible...it doesn't really matter though since nothing is found. I know TP other than those in satellites.xml exist because my crappy handheld GTMedia sat finder can see them and their channels without an issue. I can also find them if I manually scan, but these don't get saved in lamedb because they're not in satellites.xml, I assume.

![image](https://github.com/OpenPLi/enigma2-plugins/assets/414778/5b0b26c2-41f0-4293-9651-b8509f846085)

**Comment by ilikenwf** [comment]
I run into this with hardware scan for both the source version of the plugin which I currently run since I modified it to support option 1 as well as we've been discussing, but both the precompiled and the source version of the plugin result in nothing found.

I don't know what would be different between OpenATV and openpli here beyond the kernel, something with the blindscan-s2 program, or otherwise. I also noticed that dvbtraffic doesn't work for some reason either...

![image](https://github.com/OpenPLi/enigma2-plugins/assets/414778/9c2f6d18-95bc-4e7f-9240-ea28194e1c2b)


![image](https://github.com/OpenPLi/enigma2-plugins/assets/414778/8b3c1e04-e171-436a-99ff-9bd5d5fca68f)

Sorry to dump all this here. I just find that OpenPLi is much superior and lightweight compared to all the others, and would like to avoid running those if at all possible.

**Comment by Dima73** [comment]
Are the scan results the same in openATV?

**Comment by ilikenwf** [comment]
Unfortunately not. I need to get multiboot functioning properly, I think it is still on one of the partitions, but I haven't gotten around to that yet.

**Comment by ilikenwf** [comment]
I'll have to try current and v6 since I think it shipped with v6 which is very ancient...I'll flash the current openATV to slot 2 and give it a try, just a moment.

**Comment by ilikenwf** [comment]
Current 7.3 image of OpenATV - finding TP's.

![image](https://github.com/OpenPLi/enigma2-plugins/assets/414778/ce6ed5fa-306d-4da9-ad5a-7c030feb229a)

**Comment by ilikenwf** [comment]
...And this does appear to be using the hardware only scanner, not blindscan-s2. This is why I lean towards a potential difference in the kernel driver?

uname -a gives me this which isn't all that handy since I think it's the same kernel major/minorversion openpli uses.
Linux zgemmah7 4.10.12 #1 SMP Wed Apr 26 13:58:00 UTC 2017 armv7l GNU/Linux

![image](https://github.com/OpenPLi/enigma2-plugins/assets/414778/3ef57274-9186-4303-82b1-7ddc9ea5538a)

**Comment by ilikenwf** [comment]
I guess their plugin script could be somehow different too.

**Comment by Dima73** [comment]
OpenATV / OpenPLi code dmmBlindScan.py are the same on 99%

**Comment by ilikenwf** [comment]
Yes; I think openATV just uses the OE alliance version?

I can try a copy of that version if you'd like; but my gut tells me it's a kernel driver or some config thing somewhere. The Linux side of the fence is where I'm super familiar, less on the DVB side.

**Comment by Dima73** [comment]
I would suggest opening a new topic on the openPli forum.

**Comment by kueken** [comment]
It would have to work again now, so it can close it as a Issue, right?

**Comment by Dima73** [comment]
> It would have to work again now, so it can close it as a Issue, right?

Yes,close it.
